packages/ui-extensions/src/surfaces/checkout/components/Chat/examples/app-bridge-communication.example.js

```diff
@@ -3,7 +3,7 @@
 let buyerFirstName;
 
 // In the hosted application Javascript, listen for messages from the UI extension.
-shopify.extension.port.onMessage = async (event) => {
+shopify.extension.port.onmessage = async (event) => {
```

Comment:
KEEP

Reply:
@jffortier is this meant to just be part of unstable release and NOT part of 2025-04?